graphreduce/storage.py

```diff
@@ -81,7 +83,15 @@ def offload (
         elif self.compute_layer == ComputeLayerEnum.dask:
             getattr(df, f"to_{self.storage_format.value}")(self.get_path(name), index=False)
         elif self.compute_layer == ComputeLayerEnum.spark:
-            getattr(df.write, self.storage_format.value)(self.get_path(name), mode="overwrite")
+            cls_name = f"{df.__class__.__module__}.{df.__class__.__name__}"
+            if cls_name == 'pyspark.sql.connect.dataframe.DataFrame':
+                # TODO: Delete able first?
+                out_path = self.get_path(name)
+                out_path = out_path.replace('/', '.')
+                out_path = out_path.replace('.table', '')  # Take off the '.table' postfix.
+                df.write.format(self.storage_format.value).mode("append").option("mergeSchema", "true").saveAsTable(out_path)
```

Comment:
@wesmadrigal I don't know if we want to merge this yet or not. I'm still working on getting the full job running which checkpoints to tables. And I need to test more, but I suspect a delete may be required before writing the new checkpoint table.